docs/contributing/releasing-a-new-version.md

```diff
@@ -41,3 +41,4 @@ as the folder is currently ignored for performance reasons.
 There is [an issue](https://github.com/shopsys/shopsys/issues/536) on Github that mentions the problem.
 However, there is a workaround - you can add new `docker-sync` volume just for git.
 - Releasing a stage is a continuously running process so do not exit your CLI if it is not necessary.
+- Do not get a fright when you see "Version 'XY' is now released!" message after a stage is finished - this is a default behavior of `monorepo-builder` package
```

Comment:
TESTS:
maybe somewhere could be mentioned that the command needs to be run via php-fpm docker container. for example in commands
```sh
docker-compose exec php-fpm vendor/bin/monorepo-builder release <release-number> --stage <stage> -v
```

Reply:
according to the problem with unseen error - it is weird, when I test it, i can see the output correctly, see the screen...
![image](https://user-images.githubusercontent.com/10401898/50472824-b498f180-09ba-11e9-85b6-74ba8cef3578.png)
It seems like it depends on the type of the error, I am not sure what to d about it :disappointed: